Add a mention about the omission of `declare` in the cost estimation (#1908)
foundry-rs/starknet-foundry: docs/src/testing/gas-etimation.md

and gas-efficient.
`snforge` supports gas estimation for each test case. 

> 📝 **Note**
> 

> > [declare](../appendix/snforge-library/declare.md) is omitted while calculating gas.
> > This is because [declare](../appendix/snforge-library/declare.md) need to be called in separate transaction.

Reviewer: ```suggestion
> [declare](../appendix/snforge-library/declare.md) cost is omitted while calculating gas.
> This is because declare is a separate [transaction](https://docs.starknet.io/documentation/architecture_and_concepts/Network_Architecture/transactions/#declare-transaction) on Starknet.
```
